v23.1/backup.md

```diff
@@ -81,7 +81,7 @@ CockroachDB stores full backups in a backup collection. Each full backup in a co
 `LATEST` | Append an incremental backup to the latest completed full backup's subdirectory.
 <a name="collectionURI-param"></a> `collectionURI` | The URI where you want to store the backup. (Or, the default locality for a locality-aware backup.)<br/><br/>For information about this URL structure, see [Backup File URLs](#backup-file-urls).
 `localityURI`   | The URI containing the `COCKROACH_LOCALITY` parameter for a non-default locality that is part of a single locality-aware backup.
-`timestamp` | Back up data as it existed as of [`timestamp`](as-of-system-time.html). The `timestamp` must be more recent than your cluster's last garbage collection (which defaults to occur every 25 hours, but is [configurable per table](configure-replication-zones.html#replication-zone-variables)).
+`timestamp` | Back up data as it existed as of [`timestamp`](as-of-system-time.html). The `timestamp` must be more recent than your cluster's last garbage collection (which occurs at a [configurable interval](configure-replication-zones.html#gc-ttlseconds)).
```

Comment:
The same comment as below. The GC TTL does not control the interval during which we're collecting garbage, it controls how old an over-written data has to be _at least_ before it's considered for garbage collection. The garbage collection itself is actually happening continuously.

Reply:
how about

> The `timestamp` must be more recent than your cluster's garbage collection TTL (which is controlled by the [`gc.ttlseconds` replication zone variable](configure-replication-zones.html#gc-ttlseconds)).

I think that does not quite capture the nuance where something could potentially be older than `timestamp` but still work kinda by accident because something hasn't been GC'd yet but is eligible to.  But I think it might be better to just say the timestamp must be more recent.  What do you think?

Reply:
PS will port this language change to all versions v21.2+ as mentioned in another comment

Reply:
That works better, thanks. If we're being pedantic, I would change "cluster's garbage collection TTL" to "data's garbage collection TTL" since gc.ttlseconds applies over data, like specific schemas, databases, or "range default", and not the cluster.

Reply:
Yes let's be pedantic!  I have updated all versions v21.2+ to say "cluster's garbage collection TTL". Thank you!